lib/wanda/operations/operation.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Wanda.Operations.Operation do
+  @moduledoc """
+  Schema of a persisted operation.
+  """
+
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{}
+
+  @fields ~w(operation_id group_id result status agent_reports started_at completed_at)a
+  @target_fields ~w(agent_id arguments)a
+
+  @derive {Jason.Encoder, [except: [:__meta__]]}
+  @primary_key false
+  schema "operations" do
+    field :operation_id, Ecto.UUID, primary_key: true
+    field :group_id, Ecto.UUID
+
+    field :result, Ecto.Enum,
+      values: [:updated, :not_updated, :failed, :rolled_back, :skipped, :not_executed]
+
+    field :status, Ecto.Enum, values: [:running, :completed]
+
+    embeds_many :targets, Target, primary_key: false do
+      @derive Jason.Encoder
+
+      field :agent_id, Ecto.UUID, primary_key: true
+      field :arguments, :map
+    end
+
+    field :agent_reports, {:array, :map}
+
+    timestamps(type: :utc_datetime_usec, inserted_at: :started_at, updated_at: false)
```

Comment:
Why we want not to keep updated_at?

Reply:
Also could we have timestamps as the last field in the struct? I know order does not matter but ocd kicks in 👁️

Reply:
> Why we want not to keep updated_at?

We don't have it in the executions. I just copied, but well, in this case, we update the entry more often, so maybe it makes to keep it.
I will change this